MediaWikiTelegramBot.js

```diff
@@ -122,29 +155,46 @@ tgBot.onText( /\/untranslated/, function ( msg, match ) {
 
     debug( userID, 'in onText untranslated' );
 
+    // Get the messages...
     mwApi.getUntranslatedMessages( languageCode, messageCollection => {
-        debug( userID, 'in getUntranslatedMessages' );
+        // debug( userID, 'in getUntranslatedMessages' );
 
+        // Check if a user with that ID exists, creates it if not
         if ( userStatus[userID] === undefined ) {
             userStatus[userID] = {};
         }
+
+        // Set defaults
         userStatus[userID]['messages'] = messageCollection;
         userStatus[userID]['currentMwMessageIndex'] = 0;
 
-        debug( userID, 'received messageCollection ' + JSON.stringify(
-            userStatus[userID]['messages'],
-            null,
-            2
-        ) );
+        // debug( userID, 'received messageCollection ' + JSON.stringify(
+        //     userStatus[userID]['messages'],
+        //     null,
+        //     2
+        // ) );
 
         tgBot.sendMessage( userID, 'Fetched ' +
             userStatus[userID]['messages'].length + ' untranslated messages'
         );
-
-        if ( userStatus[userID]['messages'].length ) {
-            tgBot.sendMessage( userID, 'Try to translate some!' );
-            tgBot.sendMessage( userID, getCurrentMwMessage( userID ).definition );
-            mode = 'translation';
+        // var messages = userStatus[userID]['messages'];
+
+
+        // Check if the array exists and has stuff
+        if ( userStatus[userID]['messages'] && userStatus[userID]['messages'].length > 0 ) {
+            // Get the messages compiled together
+            var messages = getMessages( userID );
+
+            // Add the fluff messages
+            messages = 'Here are the messages: \n' +
+                messages +
+                '\nTry to translate some!' +
+                "\nType\n '/translate [message number] to translate that message.";
+            tgBot.sendMessage(
+                userID,
+                messages
+            )
+            // mode = 'translation';
```

Comment:
I have a new command, /translate [message number] that does it. This way, bot doesn't expect a translation for a message until user wants to translate. Chatting would still work, maybe allows for discussions? ;)